x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -442,9 +442,23 @@
     // This is for performance testing.  It's not exposed to the end user.
     // Recompile if you want to compare performance with C++ tokenization.
     public static final boolean CATEGORIZATION_TOKENIZATION_IN_JAVA = true;
-
-    public static final LicensedFeature.Persistent ML_JOBS_FEATURE =
-        LicensedFeature.persistent("machine-learning", "anomaly-detection-job", License.OperationMode.PLATINUM);
+    public static final String ML_FEATURE_FAMILY = "machine_learning";
```

Comment:
I think this should stay with a dash. I plan to rename the existing license names as they are moved out of the Feature enum.